package.json

```diff
@@ -68,6 +68,7 @@
   },
   "license": "BSD-3-Clause",
   "dependencies": {
+    "@types/unzipper": "^0.10.3",
```

Comment:
Why is this not listed as a devDependency ?

Reply:
Without it we get this:
salesforcedx-vscode-core: node_modules/@salesforce/source-deploy-retrieve/lib/src/resolve/types.d.ts(4,23): error TS7016: Could not find a declaration file for module 'unzipper'. '/Users/shelby.holden/workspace/salesforcedx-vscode/packages/salesforcedx-vscode-core/node_modules/unzipper/unzip.js' implicitly has an 'any' type.
salesforcedx-vscode-core:   Try `npm install @types/unzipper` if it exists or add a new declaration (.d.ts) file containing `declare module 'unzipper';`

Bryan and I are discussing moving a change into SDR itself to resolve this.